Day and Time/index.js

```diff
@@ -1,4 +1,25 @@
-function currentpage();
-{
-    window.print();
-}
\ No newline at end of file
+let date = new Date();
+console.log(date);
+let yr = date.getFullYear();
+console.log(yr);
+let day = date.getDay();
+var weekDays = [
```

Comment:
Lets not use var. https://codeburst.io/javascript-var-let-or-const-which-one-should-you-use-2fd521b050fa . We don't use var anymore. Lets use let and const.